kafka-ui-react-app/src/components/Topics/Topic/Messages/Message.tsx

```diff
@@ -73,7 +79,7 @@ const Message: React.FC<Props> = ({
         <td>{offset}</td>
         <td>{partition}</td>
         <td>
-          <div>{formatTimestamp(timestamp)}</div>
+          <div>{formatTimestamp(timestamp, timeStampFormat)}</div>
```

Comment:
I see no need of getting timeStampFormat in every place we use `formatTimestamp`. 

Please load `timeStampFormat` once and use inside of `formatTimestamp` helper